doc/api.rst

```diff
@@ -193,7 +195,7 @@ Benchmark class
 
       Format values including the unit.
 
-   .. method:: get_dates() -> (datetime.datetime, datetime.datetime) or None
```

Comment:
Please keep the information about the return type. Write a sentence like:

"Return (datetime.datetime, datetime.datetime) or None."